ui.tests/test-module/specs/checkboxgroup/checkboxgroup.runtime.spec.js

```diff
@@ -184,9 +184,9 @@ describe("Form Runtime with CheckBoxGroup Input", () => {
 
         const [checkBox6, checkBox6FieldView] = Object.entries(formContainer._fields)[5];
         cy.get(`#${checkBox6}`).find(".cmp-adaptiveform-checkboxgroup-item").should('have.length', 2);
-        cy.get(`#${checkBox6}`).find(".cmp-adaptiveform-checkboxgroup__label").contains('Item 3');
-        cy.get(`#${checkBox6}`).find(".cmp-adaptiveform-checkboxgroup__label").contains('Item 2');
-        cy.get(`#${checkBox6}`).find(".cmp-adaptiveform-checkboxgroup__label").contains('Item 1').should('not.exist');
+        cy.get(`#${checkBox6}`).find(".cmp-adaptiveform-checkboxgroup__option__label").contains('Item 3');
```

Comment:
BEM looks incorrect, please fix this

Reply:
Ideally this should be `.cmp-adaptiveform-checkboxgroup__option-label`

Reply:
I followed the same as provided for the radio button. Please check here https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/radiobutton.html#L40

Reply:
This needs to be changed, please fix it for radio button too. This is incorrect

Reply:
changed for both